documentation/concepts/gateway-config.md

```diff
@@ -65,6 +65,94 @@ This configuration defines a service `example-service-dev` with a corresponding
 route `example-service-route` and applies a `request-transformer` plugin to it
 to add custom headers.
 
+## Gateway configuration formats
+
+There are two Gateway configuration formats that can be used:
+
+- **Legacy format:** Introduced with v1 of the `gwa` CLI. This format supports
```

Comment:
Rather than "legacy" I think perhaps "Kong format" is more accurate - as it is still a fully supported format by Kong and us.

Reply:
True! I like that and will update accordingly